From bd89578fa3b9049a95b062c28e5b1750970dd5ed Mon Sep 17 00:00:00 2001
From: Arulselvan M <arulselvan.m@intel.com>
Date: Fri, 12 Jul 2013 07:35:53 +0530
Subject: Issues Fixed for MR2 PDK for offload Enabling

BZ: 123052

This patch fixes the Compilation issues / Offload enabling

issues while  enabling offload in MR2

Change-Id: I48768271345f5952fa395638497c4ff5b2f0c650
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
---
 include/hardware/audio_policy.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio_policy.h b/include/hardware/audio_policy.h
index 475fda6..0d11746 100644
--- a/include/hardware/audio_policy.h
+++ b/include/hardware/audio_policy.h
@@ -248,6 +248,7 @@ struct audio_policy {
                                    uint32_t samplingRate,
                                    uint32_t bitRate,
                                    int64_t duration,
+                                   int sessionId,
                                    bool isVideo,
                                    bool isStreaming);
 };
-- 
1.7.4.1

